app/api/jwt_extension.py

```diff
@@ -0,0 +1,24 @@
+from flask_jwt_extended import JWTManager
+from http import HTTPStatus
+from app import messages
+from app.api.bit_extension import api
+
+jwt = JWTManager()
+
+# This is needed for the error handlers to work with flask-restplus
+jwt._set_error_handler_callbacks(api)
+
+
+@jwt.expired_token_loader
+def my_expired_token_callback():
+    return messages.TOKEN_HAS_EXPIRED, HTTPStatus.UNAUTHORIZED
+
+
+@jwt.invalid_token_loader
+def my_invalid_token_callback(error_message):
+    return messages.TOKEN_IS_INVALID, HTTPStatus.UNAUTHORIZED
+
+
+@jwt.unauthorized_loader
+def my_unauthorized_request_callback(error_message):
+    return messages.AUTHORISATION_TOKEN_IS_MISSING, HTTPStatus.UNAUTHORIZED
```

Comment:
Should this be Unauthenticated instead?

Reply:
mmm...that's the word being used by HTTPStatus though (401 == Unauthorized)
![Screen Shot 2020-06-03 at 9 07 39 am](https://user-images.githubusercontent.com/29667122/83578333-d6b47d00-a579-11ea-9c22-f23a1f8fb989.png)

And it's also the same one used in mentorship system backend